concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAnonymityRevokers.java

```diff
@@ -0,0 +1,51 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.ClientV2;
+import com.google.common.collect.ImmutableList;
+import io.grpc.StatusRuntimeException;
+import lombok.var;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.URL;
+
+@Command(name = "GetAnonymityRevokers", mixinStandardHelpOptions = true)
+public class GetAnonymityRevokers implements Runnable {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20001")
+    private String endpoint;
+
+    @lombok.SneakyThrows
+    @Override
+    public void run() {
+        var endpointUrl = new URL(this.endpoint);
+
+        Connection connection = Connection.builder()
+                .host(endpointUrl.getHost())
+                .port(endpointUrl.getPort())
+                .credentials(new Credentials())
+                .build();
+        ClientV2 clientV2 = ClientV2.from(connection);
+        var arInfos = clientV2.getAnonymityRevokers(BlockHashInput.BEST);
+
+        try {
+            ImmutableList
+                    .copyOf(arInfos)
+                    .forEach(arInfo -> System.out.println(arInfo));
+        } catch (StatusRuntimeException ex) {
+            System.out.println(String.format("Grpc Error: %s", ex.getStatus().getCode()));
```

Comment:
Now there is a `SneakeThrows` annotation, is this `try`/`catch` clause then needed?

Reply:
@EmilLa1 - we specifically used `@SneakyThrows` because this is one of the cases where their website recommends using it. 
_"Common use cases for when you want to opt out of the checked exception mechanism center around 2 situations:
A needlessly strict interface, such as Runnable - whatever exception propagates out of your run() method, checked or not, it will be passed to the Thread's unhandled exception handler. Catching a checked exception and wrapping it in some sort of RuntimeException is only obscuring the real cause of the issue."_
https://projectlombok.org/features/SneakyThrows

However we do understand it doesn't set a nice example to be using it in a CLI application but `Runnable` doesn't allow to mention exceptions in the method signature. So we have changed the code to use `Callable` instead and have added the exceptions that can be throws in the method signature. 
https://github.com/Concordium/concordium-java-sdk/blob/3202b2cf03a13396844e34fb76d69fb6d228a012/concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAnonymityRevokers.java#L28
So that 
1. it matches closely to the sample application https://github.com/remkop/picocli#example
2. It uses the functionality provided by `picocli` where in the CommandLine executor handles the error and prints them to the console. 
https://github.com/remkop/picocli/blob/main/README.md?plain=1#L284

Any thrown error is shown in the commandline in red (color configured for an error). Ex. 
This error was synthetically generated by specifying incorrect url
``` bash
java.net.MalformedURLException: unknown protocol: httpss
	at java.net.URL.<init>(URL.java:618)
	at java.net.URL.<init>(URL.java:508)
	at java.net.URL.<init>(URL.java:457)
	at com.concordium.sdk.examples.GetAnonymityRevokers.call(GetAnonymityRevokers.java:29)
	at com.concordium.sdk.examples.GetAnonymityRevokers.call(GetAnonymityRevokers.java:19)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at com.concordium.sdk.examples.GetAnonymityRevokers.main(GetAnonymityRevokers.java:44)
```
Do you think this is sufficient for an example?
Or should be change this to manual error handling?
Please do suggest, Thanks

Reply:
My point was, that you don't need that `try/catch` clause when you have the` @SneakyThrows `annotation.

I pushed the slight change. Note. that the maven compiler plugins needs to be enabled for processing the annotations when compiling via maven.